test_data/test_semantic_parser/source/page3.txt

```diff
@@ -1 +1,29 @@
-No heading
\ No newline at end of file
+No heading
+
+.. important::
+
+   You cannot disable |global-write| for a cluster once deployed.
+
+|service| |global-write-clusters| require developers to define 
+single or multi-region :guilabel:`Zones`, where each zone supports
+write and read operations from geographically local shards. You can also
+configure zones to support global low-latency secondary reads. For more 
+information on |global-write| zones, see :ref:`global-writes-zones`.
+
+|service| does not auto-configure or auto-shard collections. 
+Sharded collections must meet specific compatibility requirements to 
+utilize |global-write|. For more information on guidance and 
+requirements for sharding collections for |global-write|, see 
+:ref:`global-writes-collections`.
+
+The |service| :ref:`data-explorer` supports creating sharded
+collections with specific validations for |global-write|. For complete
+documentation, see :ref:`de-shard-collection-for-global-writes`.
+
+.. class:: hidden
+
+   .. toctree::
```

Comment:
I think a test case with multiple `toctree` directives on a single page would be valuable!